lm_eval/utils.py

```diff
@@ -257,15 +258,15 @@ def complete_code(
                 if s[0] == tokenizer.bos_token_id:
                     s = s[1:]
                 gen_code = tokenizer.decode(
-                    s, skip_special_tokens=False, clean_up_tokenization_spaces=False
+                    s, skip_special_tokens=False, clean_up_tokenization_spaces=clean_up_tokenization_spaces
```

Comment:
This one should be checked for correctness for maintainers. Earlier was set to False, but with the flag being passed, the default behaviour is now conditioned on the flag. 

More explicitly, if a user does not supply the arg `--clean_up_tokenizations=False` the expected default behaviour of this code path changes.

Reply:
Actually most models (including replit) and tasks decoding happen here with `clean_up_tokenizations` as False because `eos_token` is in `stop_words` whenever it exists. I suggest we change the argument definition so it stores False and becomes True when called by the user. For the else scenario, default behavior will change but I don't think that impacts performance.